blog/2024-07-10-release-notes.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Luigi v2.14  
+seoMetaDescription: Release notes for Luigi v2.14
+author:
+  - Mahati Shankar
+layout: blog
+---
+
+You can read about the new features in Luigi v2.13 in the release notes below.
+
+<!-- Excerpt -->
+
+
+#### E2E Tests With Nightwatch, WebdriverIO and Puppeteer
+
+You can now use frameworks instead of Cypress which has been deprecated.
```

Comment:
Cypress is not deprecated, we just added additional testing framework support to our testing-utilities library.
Something like:
You can now use additional frameworks like Nightwatch, WebdriverIO and Puppeteer in testing-utilities library for your e2e tests.

Reply:
Changed the description as suggested.